### DISCUSSION TITLE
point_0713

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#268] [Roy]
"20190629T161839_20190629T162627_T17TMF"
![image](https://user-images.githubusercontent.com/54947195/147609268-20b64597-3e42-48c4-9d2a-0eaab8cc2f4d.png)
![image](https://user-images.githubusercontent.com/54947195/147609297-b4eabb4c-ef3f-4094-86c0-7d005302227e.png)

"20190917T161909_20190917T162652_T17TMF"
![image](https://user-images.githubusercontent.com/54947195/147609454-35f13a3f-dc05-4e6e-adbe-ee523728278d.png)
![image](https://user-images.githubusercontent.com/54947195/147609520-b98325e9-6341-4edd-a952-15ba5eb6d4cf.png)

"20200224T162149_20200224T162811_T17TMF"
![image](https://user-images.githubusercontent.com/54947195/147609710-d1d5120f-62d5-4ec7-a0aa-c7c796b7c245.png)

"20200713T161829_20200713T162814_T17TMF"
![image](https://user-images.githubusercontent.com/54947195/147610005-faea2d93-2361-4796-835f-479b50821bda.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20200224T162149_20200224T162811_T17TMF"
![Captura](https://user-images.githubusercontent.com/86502862/153461493-42be1172-2d35-46da-8a09-be9f18acf31b.JPG)
